src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java

```diff
@@ -125,5 +125,6 @@ public void onPlayerItemTakesDamage(PlayerItemDamageEvent event) {
         fire();
         event.setCancelled(cancelled);
         event.setDamage(damage.asInt());
+        event.getPlayer().updateInventory();

```

Comment:
I'll ask again... has this been tested for /both/ cancellation cases?

IE:

- Does it work right /when cancelled/?
- Dos it work right /when NOT cancelled/?